### DISCUSSION TITLE
Set month of calendar is incorrect

### DISCUSSION THREAD

**Discussion by Protonosgit** [Sketchware-Pro/Sketchware-Pro#927] [Q&A] [answered]
The set date function of the calendar component is not working when setting the month.
When set to a specific int value the result will be the set month + 1 .
(Set not Add)

**Comment by j-px** [comment, marked as answer]
While that might look like a bug, it's just how the API the Component uses was designed: https://developer.android.com/reference/java/util/Calendar#MONTH
